docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+version: "3.9"
+services:
+
+  #
+  # Portainer:
+  # Used for managing Docker containers and administration.
+  #
+  portainer:
+    container_name: portainer
+    image: portainer/portainer:latest
+    networks:
+      - darkflame_net
+    ports:
+      - "9000:9000"
+    volumes:
+      - portainer_data:/data
+      - /var/run/docker.sock:/var/run/docker.sock
+
+  #
+  # MariaDB:
+  # A port of MySQL used as the database
+  #
+  mariadb:
+    command: --default-authentication-plugin=mysql_native_password
+    container_name: mariadb
+    image: mariadb:latest
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+    networks:
+      - darkflame_net
+    ports:
+      - "3306:3306"
+    restart: always
+    volumes:
+      - db_data:/var/lib/mysql
+      - db_conf:/etc/mysql
+
+  #
+  # phpMyAdmin:
+  # A web GUI that allows you to easily access the MariaDB database
+  #
+  phpmyadmin:
+    container_name: phpmyadmin
+    depends_on:
+      - mariadb
+    environment:
+      - PMA_HOST=mariadb
+      - PMA_PORT=3306
+    image: phpmyadmin/phpmyadmin:latest
+    networks:
+      - darkflame_net
+    ports:
+      - "8080:80"
+    restart: always
+
+  # 
+  # Darkflame Universe
+  # The account manager
+  #
+  dlu-account:
+    build: https://github.com/jaller200/AccountManager.git#main
+    container_name: dlu-account
+    depends_on:
+      - mariadb
+    image: darkflame-universe-account:1.0.0
+    networks:
+      - darkflame_net
+    ports:
+      - "5000:5000"
+    volumes:
+      - "~/Desktop/Files/LU/files/account/credentials.py:/app/credentials.py"
```

Comment:
Instead of hardcoding some arbitrary path here, I'd prefer just using an empty directory in the DLU repository called `docker-config` or something where people can place all the files related to configuration. A benefit to this is that we make no assumptions about the host system and all the config will neatly be in one place. You can push empty directories to Git by placing a .gitkeep in them.

Reply:
Sounds good to me as well. What about if I put everything in the resources directory for now?

They can put the `credentials.py` and `resources.py` files in the place too if they want, or under an `account` folder there.

Reply:
Updated the file paths

Reply:
How about we use the approach of a `.env` file like I did in #36 . We could do passwords, paths, etc. all in that file and since Docker automatically reads a `.env` in the working directory into environment variables we can use all of them in the `docker-compose.yml` without having to edit it.